trade-finance-manager-ui/server/nunjucks-configuration/filter-formatUnixTimestamp.ts

```diff
@@ -0,0 +1,15 @@
+import { format, isValid, toDate } from 'date-fns';
+
+/**
+ * @param isoDateStr date string formatted as fromFormat
+ * @param toFormat date format using {@link https://date-fns.org/v3.3.1/docs/format | unicode tokens}
+ * @returns date formatted as toFormat or 'Invalid date' if can't parse isoDateStr
+ */
+export const formatUnixTimestamp = (timestamp: number, toFormat: string) => {
+  if (!timestamp) {
+    return 'Invalid date';
+  }
+
+  const date = toDate(timestamp);
+  return isValid(date) ? format(date, toFormat) : 'Invalid date';
```

Comment:
Should we use `isValidDate` from `libs/common` instead of date-fns?

Reply:
`isValid` is used in hundred of places - I dont know why `isValidDate` was added?

Reply:
Should i remove `isValidDate` and replace it with `isValid`?

Reply:
Huh, that's strange. I think @Zainzzkk recently added this

I suggest we stick with `isValid` in this instance